File: app/src/main/java/com/example/pge/models/loginApi.java
package com.example.pge.models;

public class loginApi {
    val email: String,
    val password: String
}

data class LoginResponse(
        val success: Boolean,
        val token: String?,
        val user: User?
)

data class User(
        val id: Int,
        val name: String
)

File: app/src/main/java/com/example/pge/models/LoginApi.kt
package com.example.pge.models

data class loginApi (
    val email: String,
    val password: String
)

data class LoginResponse(
    val success: Boolean,
    val token: String?,
    val user: User?
)

data class User(
    val id: Int,
    val name: String
)

